src/main/pages/che-7/installation-guide/proc_upgrading-che-using-operatorhub.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// upgrading-{prod-id-short}
+
+[id="upgrading-{prod-id-short}-using-operatorhub_{context}"]
+= Upgrading {prod-short} using OperatorHub
+
+This section describes how to upgrade from previous minor version using the Operator from OperatorHub in the OpenShift web console.
+
+.Prerequisites
+
+* An administrator account on an OpenShift instance.
```

Comment:
an OpenShift instance > an OpenShift cluster?

Not sure why we always use "instance"

Reply:
So keep it like this here for consistency. Let's address this for the whole doc later on.

Reply:
Ok, let's file an isse :D